core/templates/components/rubrics-editor/rubrics-editor.directive.ts

```diff
@@ -70,6 +73,9 @@ angular.module('oppia').directive('rubricsEditor', [
           };
 
           ctrl.saveExplanation = function(difficulty, index) {
+            if (difficulty === 'Medium' && index === 0) {
```

Comment:
Use a named constant for the difficulty (the same named constant everywhere this difficulty is used).

Reply:
Done